src/payment/strategies/stripev3/stripev3-payment-strategy.spec.ts

```diff
@@ -487,6 +487,54 @@ describe('StripeV3PaymentStrategy', () => {
             expect(response).toBe(store.getState());
         });
 
+        it('creates the order and submit payment with a signed user without a phone number provided', async () => {
+            const elements = stripeV3JsMock.elements();
+            const cardElement = elements.create(StripeElementType.CreditCard, {});
+
+            const customer = {
```

Comment:
You still have not apply the suggested change.

Reply:
You're right, this change was addressed.